.github/workflows/release.yml

```diff
@@ -34,6 +36,9 @@ jobs:
           NPM: true
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           RALLY_API_KEY: ${{ secrets.RALLY_API_KEY }}
+      - name: Get new git HEAD
+        id: git
+        run: echo "head=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

Comment:
This will be different then `GITHUG_REF` if semantic release generated and pushed a new commit